.test-infra/metrics/grafana/dashboards/github_actions_post-commit_tests.json

```diff
@@ -0,0 +1,557 @@
+          },
+          "mappings": [
+            {
+              "options": {
+                "cancelled": {
+                  "color": "super-light-yellow",
+                  "index": 2
+                },
+                "failure": {
+                  "color": "dark-red",
+                  "index": 1
+                },
+                "success": {
+                  "color": "green",
+                  "index": 0
+                }
+              },
+              "type": "value"
+            }
+          ],
+          "noValue": "Fetching status",
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Workflow URL"
+            },
+            "properties": [
+              {
+                "id": "custom.displayMode",
+                "value": "auto"
+              },
+              {
+                "id": "custom.width",
+                "value": 410
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Created At"
+            },
+            "properties": [
+              {
+                "id": "custom.displayMode",
+                "value": "auto"
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 12,
+        "y": 8
+      },
+      "id": 4,
+      "options": {
+        "showHeader": true,
+        "sortBy": []
+      },
+      "pluginVersion": "8.0.4",
+      "targets": [
+        {
+          "cacheDurationSeconds": 300,
+          "fields": [
+            {
+              "jsonPath": "$.workflow_runs[*].id",
+              "name": "Workflow ID"
+            },
+            {
+              "jsonPath": "$.workflow_runs[*].conclusion",
+              "name": "Test Result"
+            },
+            {
+              "jsonPath": "$.workflow_runs[*].html_url",
+              "name": "Workflow URL"
+            },
+            {
+              "jsonPath": "$.workflow_runs[*].created_at",
+              "name": "Created At"
+            }
+          ],
+          "method": "GET",
+          "queryParams": "",
+          "refId": "A",
+          "urlPath": ""
+        }
+      ],
+      "title": "Java Workflow Results",
+      "type": "table"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 30,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-24h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "GitHub Actions Post Commit Tests Copy",
+  "uid": "BsovMmVnk",
+  "version": 1
+}
```

Comment:
nit - can we add a newline here?

Reply:
> nit - can we add a newline here?

Totally! I will the modified version in a few.
Thanks for the heads up!